Example/Analytics_Tests_iOS/Analytics_Tests_iOS.m

```diff
@@ -21,17 +21,19 @@ @interface Analytics_Tests_iOS : XCTestCase
 @implementation Analytics_Tests_iOS
 
 - (void)setUp {
-    [super setUp];
-    // Put setup code here. This method is called before the invocation of each test method in the class.
+  [super setUp];
```

Comment:
(No action needed) Comment for the future: If we're going to leave all these empty, is there a point in having them here? Not that it makes a huge difference though.

Reply:
Agreed.  The teams currently have different style guidelines, so we'll need a bit more coordination. 

The Analytics tests are hopefully temporary until we have the ability to do pod testing closer to its development.

Thanks for the review!